pkg/graveler/graveler_v2_test.go

```diff
@@ -597,6 +597,26 @@ func TestGravelerCommit_v2(t *testing.T) {
 	})
 }
 
+func TestGravelerCreateCommitRecord_v2(t *testing.T) {
+	ctx := context.Background()
+	t.Run("create commit record", func(t *testing.T) {
+		test := testutil.InitGravelerTest(t)
+		commit := graveler.Commit{
+			Committer:    "committer",
+			Message:      "message",
+			MetaRangeID:  "metaRangeID",
+			Parents:      []graveler.CommitID{"parent1", "parent2"},
+			Metadata:     graveler.Metadata{"key": "value"},
+			CreationDate: time.Now(),
+			Version:      graveler.CommitVersion(2),
```

Comment:
```suggestion
			Version:      graveler.CurrentCommitVersion,
```
(and elsewhere, too!)

- 2 is not a valid value, only 0 and 1 are valid
- using the symbolic name makes sense here